src/generic/delete-modal/DeleteModal.jsx

```diff
@@ -9,13 +9,18 @@ import { useIntl } from '@edx/frontend-platform/i18n';
 import messages from './messages';
 
 const DeleteModal = ({
-  category, isOpen, close, onDeleteSubmit,
+  category,
+  isOpen,
+  close,
+  onDeleteSubmit,
+  title,
+  description,
 }) => {
   const intl = useIntl();
 
   return (
     <AlertModal
-      title={intl.formatMessage(messages.title, { category })}
+      title={title || intl.formatMessage(messages.title, { category })}
```

Comment:
[question]: Can we create a variable for this?

Reply:
Fixed